control/veda_deliver_xuetang.py

```diff
@@ -1,197 +0,0 @@
-
-import os
-import hashlib
-import hmac
-import base64
-import datetime
-import requests
-import json
-import time
-from time import strftime
-from VEDA.utils import get_config
-
-"""
-Authored by Ed Zarecor / edx DevOps
-
-included by request
-
-Some adaptations for VEDA:
-    -auth yaml
-
-   **VEDA Note: since this isn't a real CDN, and represents the
-    'least effort' response to getting video into china,
-    we shan't monitor for success**
-
-
-"""
-auth_dict = get_config()
-
-API_SHARED_SECRET = auth_dict['xuetang_api_shared_secret']
-API_ENDPOINT = auth_dict['xuetang_api_url']
```

Comment:
@greg can we also remove this from instance_config ?

Reply:
@muzaffaryousaf That't the wrong handle. And yes, I've got it removed in https://github.com/edx/edx-video-pipeline/pull/55